plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -388,6 +388,10 @@ module.exports = class ServiceTree extends Tree {
 
   getTaskCount() {
     return this.flattenItems().getItems().reduce(function(memo, service) {
+      if (service instanceof ServiceTree) {
```

Comment:
`#flattenItems` does flatten it into a single array, but the array also includes the `ServiceTree` instances, so the accumulated total was incorrect.